src/remove.js

```diff
@@ -105,23 +105,15 @@ export default function remove(path, globalOptions, options) {
         break
 
       case 'declarator':
-        if (mode === 'unsafe-wrap') {
```

Comment:
For posterity, the reason we decided that the wrap behavior is a good choice for the unsafe-wrap mode here is because const and let are block-scoped, and we don't want to change the scoping or semantics of the original code.